### ISSUE TITLE
Feature request: exclude files from import job

### ISSUE THREAD

**Issue by thomasheartman** [enonic/site-developer#382]
When working with multi-document guides, it's very handy to be able to re-use variables across files. Asciidoc allows you to do this by storing them in a file that contains only variables (which can then be imported into other files).

However, because all files are imported by default, this file also shows up as an empty page in the guide.

To exclude the variables file (and to allow other files to exist in the tree without being imported), it has been suggested to add an exclusion pattern. The leading suggestion is to not import files whose name starts with a dot, e.g. `.variables.adoc`.

**Comment by sigdestad** [comment]
FYI, the import handles the compiled html files only. Is there a way to exclude compliation of certain adoc files maybe?

**Comment by thomasheartman** [comment]
Turns out that this feature already exists: files with a leading `.` get ignored, so in my case, I renamed the file from `variables.adoc` to `.variables.adoc`. This appears to make the system not build this page. As such, the feature request is granted and the issue can be closed.

However, it seems that no-one was aware of this feature, so I'm guessing it's not documented anywhere. Is there a place where this can be documented? I've been thinking about writing down some of the things I've learned about the process and the system while I've been working on it, so I could put it there, but it doesn't matter much if no-one ever finds it 🤔
